<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 134, 25);
QMenu::sig_in
{
background-color: rgb(48, 178, 40);
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>362</width>
      <height>252</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="QTextEdit" name="commonChatBrowser">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(181, 233, 22);

</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="sendMessage">
       <property name="styleSheet">
        <string notr="true">border-bottom-color: rgb(181, 233, 22);
</string>
       </property>
       <property name="text">
        <string>Отправить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="messageLineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="placeholderText">
        <string>Введите сообщение</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>383</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Авторизация</string>
    </property>
    <addaction name="sig_in"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="sig_in">
   <property name="text">
    <string>Ввести имя</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
